client/src/component/NavBar/NavBar.jsx

```diff
@@ -1,17 +1,33 @@
-import React from 'react';
+import React, {useState, useEffect} from "react";
+import {useSelector} from 'react-redux'
```

Comment:
Really interesting hook that will be useful in our functional components when trying to access global state